jobs/silk-cni/spec

```diff
@@ -112,3 +112,7 @@ properties:
     description: |
       EXPERIMENTAL: Maximum number of outbound connections to be opened per second per port on destination host per container given that the burst is exhausted.
       Has no effect when `outbound_connections.limit` is false.
+
+  outbound_connections.dry_run:
+    description: "Introduce the options of a dry-run mode. Default: false"
```

Comment:
This is just so self-descriptive... You don't bring any value by commenting this. Just mention what it should do, i.e. that no actual rate-limiting of outbound connections would be applied and only logging of when a connection would be dropped is going to be done. Specifying the default value is useless.